ts/Core/Pointer.ts

```diff
@@ -1233,6 +1233,14 @@ class Pointer {
         const chart = charts[pick(H.hoverChartIndex, -1)];
         const tooltip = this.chart.tooltip;
 
+        if (tooltip && tooltip.container) { // #14434
+            const rect = tooltip.container.getBoundingClientRect();
+            if (e.clientX >= rect.left && e.clientX <= rect.right &&
+                e.clientY >= rect.top && e.clientY <= rect.bottom) {
+                return;
+            }
+        }
+
```

Comment:
I was hoping for a simpler check, something like this:
```js
if (tooltip && tooltip.isStickyOnContact()) {
    return;
}
```

But it doesn't work. The internal `tooltip.inContact` flag is false, that should work here. Can you have a look at why that doesn't work?

Reply:
I think its because the tooltip label is smaller than the container, and the handler that sets inContact is attached to the label.

Reply:
Or its not really about size since the charts `onContainerMouseLeave` is going to fire first anyway.

Reply:
Hmm. In that case we can use something like `if (this.inClass(relatedTarget, 'highcharts-tooltip-container'))` => return.

@bre1470 Any suggestions on how we can avoid having double logic to detect whether to stick on contact?

Reply:
Using `setTimeout(,0)` in `onContainerMouseLeave` could work.

Reply:
That sets `pointerEvents` to `auto` when `stickOnContact` is `true`, otherwise the handlers that set `inContact` wouldnt fire at all. And its probably `none` when `stickOnContact` is `false` so that you can click through the tooltip.